sdk/core/core-lro/samples-dev/customPoller.ts

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary demnostrates how to create a custom poller.
+ */
+
+import type { OperationState, OperationStatus, PollerLike } from "@azure/core-lro";
+import { delay } from "@azure/core-util";
+
+const DEFAULT_POLL_INTERVAL_IN_MS = 2000;
+
+type OperationConfig = { id: string };
+
+type RestorableOperationState<TResult, TState extends OperationState<TResult>> = TState & {
+  config: OperationConfig;
+};
+
+async function initOperation<T>(): Promise<RestorableOperationState<T, OperationState<T>>> {
```

Comment:
Same here, I think we need to provide more information about what these operations represent and how they are to be used.

Reply:
Good point, addressed in https://github.com/Azure/azure-sdk-for-js/pull/32469/commits/9babca3eabce977777fdbb2aba1959735d48fcd2